### PR TITLE
Camino refactor of subnet-cli tool

### PR DESCRIPTION
**Pull request by pnowosie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,25 +70,6 @@ Flags:
...
---ledger \
---node-ids=NodeID-741aqvs6R4iuHDyd1qT1NrFTmsgu78dc4,NodeID-K7Y79oAmBntAcdkyY1CLxCim8QuqcZbBp,NodeID-C3EY6u4v7DDi6YEbYf1wmXdvkEFXYuXNW,NodeID-AiLGeqQfh9gZY3Y8wLMD15tuJtsJHq5Qi \
---vm-genesis-path=fake-genesis.json \
---vm-id=tGas3T58KzdjLHhBDMnH2TvrddhqTji5iZAMZ3RXs2NLpSnhH \
---chain-name=test
-```
-
-_Make sure you've downloaded the latest version of the
-[Avalanche Ledger App](https://docs.avax.network/learn/setup-your-ledger-nano-s-with-avalanche)!_
-
```
Comment (DerTiedemann) [inline README.md:91]: why remove ledger support?

File: README.md [hunk 2]
```diff
@@ -105,17 +86,16 @@ subnet-cli create VMID <identifier> [--hash]
 subnet-cli create key
 ```
 
-`subnet-cli` will assume you have funds on this key (or `--private-key-path`) on the P-Chain for the
-rest of this walkthrough.
+This creates a file `.subnet-cli.pk` under the current directory with a private key. By default,
+`subnet-cli` uses the key specified in file `.subnet-cli.pk` on the P-Chain to pay for the transaction fee, unless `--private-key-path` is used to overwrite. Please make sure that you have enough fund on this P-Chain address to pay for transactions.
```
Comment (DerTiedemann) [inline README.md]:
```suggestion
`subnet-cli` uses the key specified in file `.subnet-cli.pk` on the P-Chain to pay for the transaction fee, unless `--private-key-path` is used to overwrite. Please make sure that you have enough funds on this P-Chain address to pay for transactions.
```

File: README.md (hunk 1 above)
Comment (pnowosie) [reply README.md:91]: It's made of `github.com/ava-labs/avalanche-ledger-go` sdk which seems to work only with Avalanche wallet.
One thing I should do better is to make it clear to the user and write some message when `--ledger` flag is used

Comment (pnowosie) [reply README.md:91]: Ledger removed completely in 0b45dd25d647d0df62e00d82d638bc47cb20a4d8

**Review by DerTiedemann — APPROVED** [review]
LGTM

**Pull request merged**